### PR TITLE
Added a bot for contributers

### PR DESCRIPTION
**Pull request by shivamvishwakarm**
### Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Checklist:

- [x] This pull request follow our contribution guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (postman and swagger)
- [ ] I have written tests for the code
- [ ] I have updated .env.example file for new .env variables

### What is current behavior?

Closes #(issue_number)

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "projectName": "MultiEmail-backend",
+  "projectOwner": "(Ayush Chugh)",
```
Comment (aayushchugh) [inline .all-contributorsrc]: I am not only owner so make owner `Multi Email`

File: readme.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Technologies Used
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```
Comment (aayushchugh) [inline readme.md:3]: Move the badge above `Technologies Used` section

File: .all-contributorsrc (hunk 1 above)
Comment (aayushchugh) [inline .all-contributorsrc]: I am not only owner of the project so change `projectOwner` field to `Multi Email`

**Review by aayushchugh — CHANGES_REQUESTED** [review]
Also remove `Developers/Contributors` Section because you have create a `Contributors` section

image

**Comment by aayushchugh** [comment]
@shivamvishwakarm please follow contribution guidelines and follow conventional commits for writing commit messages
and also update the pull request description to actually what you did instead of of just submitting our pre defined template

I have created pull request template so that you can take refrence from it

File: readme.md [hunk 3]
```diff
@@ -56,3 +62,15 @@ This project follows the [all-contributors](https://github.com/all-contributors/
...
+
+## Contributors ✨
+
+Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
```
Comment (aayushchugh) [inline readme.md:74]: Please add ALL-CONTRIBUTORS-LIST:END line

File: readme.md [hunk 4]
```diff
@@ -1,4 +1,10 @@
 # Technologies Used
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```
Comment (aayushchugh) [inline readme.md:3]: Please remove this dublicate

**Review by aayushchugh — CHANGES_REQUESTED** [review]

File: readme.md (hunk 3 above)
Comment (aayushchugh) [inline readme.md:73]: Add ALL-CONTRIBUTORS-LIST:START here

**Review by aayushchugh — CHANGES_REQUESTED** [review]

**Review by aayushchugh — APPROVED** [review]

**Comment by aayushchugh** [comment]
nice

**Comment by shivamvishwakarm** [comment]
Thank's for your support aayush. you instruct very nicely

**Comment by aayushchugh** [comment]
@all-contributors please add @shivamvishwakarm for doc

**Pull request merged**